cancermuts/datasources.py

```diff
@@ -232,8 +232,10 @@ def _get_aliases(self, gene_id, to, fr='UniProtKB_AC-ID'):
             if len(responses) == 0:
                 self.log.warning(f"No {t} found for {gene_id}")
                 return None
+
             if len(responses) > 1:
-                raise TypeError
+            	self.log.warning(f"Multiple {t} found for {gene_id}. Entrez ID processing and mutation retrieval from cBioPortal will be skipped. Found {t}: {[response['to'] for response in responses]}")
```

Comment:
can you please
* remove the sentence that says that cBioPortal will be skipped, since this is not in the right place for it
* can the responses be comma-separated in the warning? e.g. `asd, qwe`. You can `', '.join()` for that
* can you also state that no identifier will be assigned for that type? (e.g. no Entrez ID will be assigned)